src/test/java/uk/gov/companieshouse/model/validator/EndDateValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.TestData;
+import uk.gov.companieshouse.TestUtil;
+import uk.gov.companieshouse.model.Appeal;
+
+public class EndDateValidatorTest
```

Comment:
Remove **public** to please Sonar. There's some other sonar issues that need sorting